Formula/dinghy.rb

```diff
@@ -4,6 +4,9 @@ class Dinghy < Formula
   url "https://github.com/codekitchen/dinghy.git", :tag => "v4.6.3", :revision => "05349a35ee9858dce3497ce0b90eb5f2dbb1bca8"
   head "https://github.com/codekitchen/dinghy.git", :branch => :master
 
+  # Dinghy version hasn't changed, but unfs3 dependency has
+  revision 1
```

Comment:
I'm not actually super familiar with writing Homebrew Formulas -- was the idea with adding this line that it would force Homebrew to install the new version of unfs3 when upgrading Dinghy? It doesn't seem to have done that in my testing just now.

I'm looking through the Homebrew docs trying to find a way to ensure that when people `brew upgrade dinghy` it'll upgrade unfs3 as well but I'm not seeing anything yet. I'll be bummed if our only option is to tell people to `brew upgrade codekitchen/dinghy/unfs3` too but maybe that's the case.

Reply:
I spent a little time yesterday trying to find a way to specify a version for the dependency or force an upgrade but couldn't find one. It's not clear from the Homebrew documentation how this sort of dependency is handled. :/

Reply:
Yeah I don't see a better option, just seems silly to not be able to force it.